src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileFormat.cs

```diff
@@ -245,7 +244,7 @@ private static LockFile ReadLockFile(JObject cursor, string path, LockFileReadFl
                 : new List<CentralTransitiveDependencyGroup>();
 
             var logMessage = (flags & LockFileReadFlags.LogMessages) == LockFileReadFlags.LogMessages
-                ? ReadLogMessageArray(cursor[LogsProperty] as JArray, packagesSpec?.RestoreMetadata?.ProjectPath)
+                ? ReadLogMessageArray(cursor[CacheFileProperties.LogsProperty] as JArray, packagesSpec?.RestoreMetadata?.ProjectPath)
```

Comment:
I'm not sure I see why it should belong in the CacheFileProperties instead of the LockFileFormat. It's already internal here,. No need to make changes in this file.

Reply:
From the perspective of CacheFile sure, but from the perspective of the LockFile, it's doing the exact opposite, now the lock file is referencing the cache file. 

I think both of our approaches are a bit opinionated, but there's a precedent here and we're replacing it with an equivalent one.